src/components/Tabs/components/TabPanel/TabPanel.tsx

```diff
@@ -0,0 +1,26 @@
+import { Tab } from "../../types"
+import styles from "./TabPanel.module.css"
+
+interface TabPanelProps {
+  tab: Tab
+}
+
+export function TabPanel({ tab }: TabPanelProps) {
+  const ariaHidden = tab.isSelected === false
+  const className = `${styles.TabPanel}${
+    ariaHidden ? ` ${styles.isHidden}` : ""
+  }`
```

Comment:
Great question. @albatrocity had suggested that we might draw the line on dependencies and exclude the usage of libs like `classnames` and instead manually do this sort of thing—so since this case was pretty simple I went for it manually, BUT haven't gone back and reworked the spot where I'm still using `classnames`. What are your thoughts—is it worth trying to avoid using it?

Reply:
I've removed the dependency on `classnames` from `TabList.tsx` as well.